experimental/cloudstrapper/playbooks/roles/build-platform/files/helm-publish.bash

```diff
@@ -0,0 +1,16 @@
+#arg1: gitusername arg2: gitpassword arg3: git publish branch 
+gitBranchCheckout=a7580153
```

Comment:
What does this branch correspond to?

Reply:
This is from our 1.3 install config where the process was slightly different than 1.4. This code can be removed once we release 1.5 since we would then have only 1.4 and 1.5 as supported deployment versions via Cloudstrapper.